code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -360,11 +384,19 @@
 		to_chat(src, "<span class='userdanger'>You are still bound to serve the cult, follow their orders and help them complete their goals at all costs.</span>")
 	else
 		to_chat(src, "<span class='userdanger'>You are still bound to serve your creator, follow their orders and help them complete their goals at all costs.</span>")
+	var/atom/movable/body = SS.get_body()
+	body.forceMove(src)
 	cancel_camera()
 	qdel(shell)
 	qdel(shade)
 	qdel(SS)
 
+/obj/item/soulstone/proc/get_body()
```

Comment:
I'd prefer to have a field on the `soulstone` which saves the body. You'd still need to forceMove the body into the stone. But having a reference just makes the code more clear. Just be sure to also handle GC properly (clearing a reference when removed etc)

Reply:
add a variable for this